terraform/variables.tf

```diff
@@ -164,6 +164,28 @@ variable "env_templated_site_name" {
   default = "MyNotif"
 }
 
+variable "env_stripe_public_key" {
+  type        = string
+  description = "Stripe API Key"
+  default     = "pk_live_51QJYlSKlp91vayS3DNT5o4UovOaGsO5hYFHB6HwD9xzcRvDvjP7v7NuAXv55IA9r9o8rr7HhERITYtBmgDI9MXR600lecyFhm4"
+}
```

Comment:
I'm not even sure this is really used in the backend in fact, maybe revisit the backend code first and see if this is really needed. I know we have a reference to that environment variable in the backend, but I'm not sure the stripe library is actually consuming it, can you double check?

Reply:
I delete it because isn't use

Reply:
Thanks, can you also make a PR backend side removing that environment variable so we can later rebase